en/docs/deploy/aws.md

```diff
@@ -0,0 +1,88 @@
+# Deploying WSO2 Identity Server on Kubernetes using Minikube
+
+!!! info "Prerequisites"
+
+    Install the following applications if you do not have them installed already. Make sure you install the recommended versions for a seamless deployment. 
+
+    1.  Install [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git). 
+    
+    2.  Install [Helm](https://v2.helm.sh/docs/using_helm/#installing-helm) and [Tiller](https://v2.helm.sh/docs/using_helm/#installing-tiller) **version 2.9.1**.
+
+    3.  Install [Kubernetes Client](https://kubernetes.io/docs/tasks/tools/install-kubectl/) **version-v1.17.3**. 
```

Comment:
The prerequisites up to this point seem to be the same in all the deployment docs...Shall we include this part in a fragment.md file inside the "fragments" folder and include it in all the relevant docs so that it is not duplicated?

Reply:
Different platforms work with different versions. Ideally, they should all work with the latest versions. We can't use a single template until that happens.